geonode/client/templates/ol/layers/layer_ol2_map.html

```diff
@@ -13,7 +13,7 @@
 <script type="text/javascript">
   document.addEventListener("DOMContentLoaded", function (event) {
     {% if resource.ll_bbox %}
-      var bbox = [{{ resource.ll_bbox_string }}];
+      var bbox = [{{ resource.bbox_string }}];

```

Comment:
The bbox here should always be returned into 4326 SRS. Please, revert to ll_bbox_string.